### PR TITLE
Add E2E test scripts

### PR DESCRIPTION
**Pull request by TylorShin**

### CODE REVIEW

File: e2eTest/index.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import MainPageTester from "./mainPageTester";
+import SearchResultTester from "./searchResultTester";
+import SearchTester from "./searchTester";
+
+export interface E2ETestManager {
+  test: () => Promise<void>;
+}
+
+class E2EManager {
```
Comment (am2rican5) [inline e2eTest/index.tsx]: How about let the manager collect testers to visualize their test result (with error handler)?

Comment (TylorShin) [reply e2eTest/index.tsx]: I can't get what u mean now.
Do u wanna say logs(messages) or progress?

Comment (am2rican5) [reply e2eTest/index.tsx]: I mean, displaying statistical result like other test runners would be better.

Comment (TylorShin) [reply e2eTest/index.tsx]: I made it very roughly. At the current level, I think it costs too many time.

**Comment by am2rican5** [comment]
👍

**Pull request merged**